src/api/app.py

```
@@ -1,20 +1,29 @@
-from flask import Flask, jsonify, request, send_from_directory
+from flask import Flask, jsonify, request, send_from_directory, session, abort
```

Ruff (0.11.9)

`flask.send_from_directory` imported but unused

Remove unused import: `flask.send_from_directory`

(F401)